glsl.h

```diff
@@ -0,0 +1,23 @@
+#ifndef GLSL_H
+#define GLSL_H
+
+#include "vectorMath.h"
+#include "noise.h"
+
+using namespace vm;
+
+namespace GLSL
+{
+    static Noise glslNoise(0, 0.02, 1);
```

Comment:
Might be best to not have a global noise and let the shaders sample their own noise with their own parameters rather than predefining a noise.